controllers/recipesController.js

```diff
@@ -8,10 +9,57 @@ module.exports = {
       .then((dbModel) => res.json(dbModel))
       .catch((err) => res.status(422).json(err));
   },
-  create: function(req, res) {
-      db.Recipe
-        .create(req.body)
-        .then(dbModel => res.json(dbModel))
-        .catch(err => res.status(422).json(err));
-  }
+  create: function (req, res) {
+    addIngredients(req.body.ingredients).then((ingredients) => {
```

Comment:
I guess the main reason for this is to create the ingredients and tags before the recipe which references them. Makes sense.